x-pack/platform/plugins/shared/alerting/server/routes/index.ts

```diff
@@ -158,4 +159,67 @@ export function defineRoutes(opts: RouteOptions) {
   runSoonRoute(router, licenseState);
   healthRoute(router, licenseState, encryptedSavedObjects);
   getGlobalExecutionKPIRoute(router, licenseState);
+
+  router.post(
```

Comment:
we should move these routes that look like they're for testing to the plugin setup for one of the functional tests